webtester-core/src/test/java/integration/pageobjects/SingleSelectIntegrationTest.java

```diff
@@ -0,0 +1,187 @@
+package integration.pageobjects;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.NoSuchElementException;
+
+import integration.AbstractWebTesterIntegrationTest;
+
+import info.novatec.testit.webtester.api.annotations.IdentifyUsing;
+import info.novatec.testit.webtester.api.exceptions.WrongElementClassException;
+import info.novatec.testit.webtester.pageobjects.PageObject;
+import info.novatec.testit.webtester.pageobjects.SingleSelect;
+
+
+
+public class SingleSelectIntegrationTest extends AbstractWebTesterIntegrationTest {
```

Comment:
oh okay, but even `matchers.isEmptyString()` does not work, because the String is null and not empty. But if i test it with `nullValue()` there is an exception too.

I found the `Matchers.nullValue()` before, but both versions does not work with the code either